### ISSUE TITLE
WebRTC low quality audio after RTMP->RTC transcoding

### ISSUE THREAD

**Issue by streamthing** [ossrs/srs#2673]
**描述(Description)**
RTMP -> WebRTC transcoding provides low audio quality.
There is no option to change opus bitrate in rtmp_to_rtc function.
I provide 256kbps AAC audio via RTMP. Diffrence between RTMP and WebRTC is hearable.

1. SRS版本(Version): `SRS/4.0.177(Leo) - Docker build`
1. SRS的配置如下(Config):
```
rtc_server {
    enabled on;
    listen 8000;
    candidate $CANDIDATE;
}

vhost __defaultVhost__ {
    rtc {
        enabled     on;
        rtmp_to_rtc on;
        rtc_to_rtmp off;
    }
}
```

**重现(Replay)**
1. `Open music RTMP Stream`
2. `Open music WebRTC Stream`
3. `Compare between RTMP and WebRTC streams`

**期望行为(Expect)**
Option to change opus bitrate or best audio quality.

**Comment by winlinvip** [comment]
因为SRS转码AAC到OPUS，码率更低了。

质量达不到要求么？

**Comment by winlinvip** [comment]
SRS transcode AAC to OPUS, which is low bitrate.

So the quality is not good enough? Please attach an example file?

**Comment by streamthing** [comment]
Opus can provide max bitrate ~450kbps.
We need at least 192kbps for Opus.
Audio quality is very poor with 64kbps.

Configuration should be able to change the bitrate depending on the application (voice / music)

Here you can setup bitrate of Opus transcoding - https://github.com/AirenSoft/OvenMediaEngine/blob/master/misc/conf_examples/Server.xml#L250

**Comment by streamthing** [comment]
@winlinvip 
I tested and can confirm that changing this parametres in trunk/src/app/srs_app_rtc_codec.cpp
```
Line 239: enc_->bit_rate = 160000; (160Kbps)
Line 244: enc_->compression_level = 8;
```
Solves problem with bad audio.
Could you add this options to change in .conf file?

**Comment by xiaozhihong** [comment]
> @winlinvip I tested and can confirm that changing this parametres in trunk/src/app/srs_app_rtc_codec.cpp
> 
> ```
> Line 239: enc_->bit_rate = 160000; (160Kbps)
> Line 244: enc_->compression_level = 8;
> ```
> 
> Solves problem with bad audio. Could you add this options to change in .conf file?

Good idea, using config solve it.

**Comment by winlinvip** [comment]
Fixed by #3515
